bindings/jaxrs/src/main/java/org/commonjava/indy/core/bind/jaxrs/ContentAccessHandler.java

```diff
@@ -172,6 +177,54 @@ public Response doDelete( final String packageType, final String type, final Str
         return doDelete( packageType, type, name, path, eventMetadata, null );
     }
 
+    public Response doDelete( final String packageType, final String type, final String name,
```

Comment:
Are all these (packageType, type and name) necessary here? They are all in the DemoteRequest. I would either remove these 3 params or replace the request by simple List<String>.
 BTW also the trackingID from the DemoteRequest is not used at all. Not sure what the intended purpose was.

Reply:
Yeah, it looks a bit confusing, the trackingID is used in the folo side. And why the 3 params are here is because they are parts of the parent rest api path, so if we use this api not folo one, just specify the list works. 
Anyway, i will try to figure out how to refactor it to make it simple.